std_msgs/Header header
ThrotStat throt_stat_msg
ThrotDiag throt_diag_msg
BrakeStat brake_stat_msg
BrakeDiag brake_diag_msg
ParkStat park_stat_msg
ParkDiag park_diag_msg
SteerStat steer_stat_msg
SteerDiag steer_diag_msg
BodyStat body_stat_msg
BodyDiag body_diag_msg
AnticrshStat anticrsh_stat_msg
BatStat bat_stat_msg
BatDiag bat_diag_msg
VehStat veh_stat_msg
VehDynaStat veh_dyna_stat_msg
WheelSpdStat front_wheel_spd_stat_msg
WheelSpdStat rear_wheel_spd_stat_msg
PowerStat power_stat_msg
TripStat total_trip_msg
TripStat remote_trip_msg
TripStat automatic_trip_msg
TripStat cloud_trip_msg
RemoterStat1 remoter_stat1_msg
RemoterStat2 remoter_stat2_msg
RemoterStat3 remoter_stat3_msg
VehInfo veh_info_msg
#imu
ImuStat imu_stat_msg